### ISSUE TITLE
License request

### ISSUE THREAD

**Issue by ccorbella** [Jwomers/many-time-pad-attack#1]
Dear @Jwomers,

I would like to use your code as a part of a little program. Could you provide a license so that I can do so (https://choosealicense.com/)? 

Regards,
Conrad

**Comment by Jwomers** [comment]
Hi @ccorbella what do you want to use this for?

Thanks, 
Justin

**Comment by ccorbella** [comment]
Hi,

I would like to use it for a little part (an example) in nonprofit institution open educational program created for teaching purposes, just to show why one time pad must be **one time**. xD

The user (student) would be allowed to encode a few messages and to recover part of them using your technique -searching for spaces-, looking for a few common words else and checking up the partial words in a dictionary. That way they would find out that it reusing keys makes that encryption breakable.

If for some reason, you preferred to keep your code without license, then I would not use it and I would fully write it myself, as in fact I was doing until I found yours and took it as a basis.

Thanks,
Conrad

PD. Sorry for opening an issue -I did not know how to reach you, otherwise- and in the webpage I sent you it was said that this was the best way.
